.github/workflows/test-sample-apps.yml

```diff
@@ -56,6 +56,8 @@ jobs:
           cf8 --version
       - name: Build Java apps
         run: ./http2/gradlew clean build -p http2
+      - name: Remove routes from manifest
+        run: yq 'del(.applications[].routes)' -i "${GITHUB_WORKSPACE}/http2/apps-manifest.yml"
```

Comment:
instead of deleting, could we define a (default empty) prefix variable?

e.g. 
```yaml
application:
  route: "((prefix))some-app-name.((domain))"
```

and then define `prefix: ""` in the vars.yml, to override it when deploying?